src/main/java/emissary/directory/EmissaryNode.java

```diff
@@ -71,6 +74,16 @@ public class EmissaryNode {
     protected boolean nodeNameIsDefault = false;
     protected String nodeServiceType = null;
 
+    public boolean isStartUpMode() {
```

Comment:
I think a comment indicating what "startUpMode" is would be helpful. Is it to indicate that this specific place fails, the whole startup should fail?